rules-tests/Transform/Rector/Class_/AddAllowDynamicPropertiesAttributeRector/config/configured_rule.php

```diff
@@ -7,5 +7,10 @@
 
 return static function (ContainerConfigurator $containerConfigurator): void {
     $services = $containerConfigurator->services();
-    $services->set(AddAllowDynamicPropertiesAttributeRector::class);
+    $services->set(AddAllowDynamicPropertiesAttributeRector::class)
+        ->call('configure', [[
+            AddAllowDynamicPropertiesAttributeRector::TRANSFORM_ON_NAMESPACES => [
+                '*\Fixture\Process\*'
+            ]
+        ]]);
```

Comment:
can be changed to:

```suggestion
        ->configure(
            [
                '*\Fixture\Process\*'
            ]
        );
```

Reply:
We should have probably PHPStan rule for this one :+1: 

In https://github.com/rectorphp/phpstan-rules